yang/frr-route-map.yang

```diff
@@ -179,17 +179,27 @@ module frr-route-map {
                 description "Match a route tag";
                 value 10;
               }
-
-              /*
-               * Protocol YANG models should augment the parent node to
-               * contain the routing protocol specific value. The protocol
-               * must also augment `condition-value` to include its specific
-               * values or expand the `when` statement on the existing cases.
-               */
-              enum routing-protocol-specific {
-                description "Match a routing protocol specific type";
+              /* zebra specific conditions. */
+              enum ipv4-prefix-length {
+                description "Match IPv4 prefix length";
                 value 100;
               }
+              enum ipv6-prefix-length {
+                description "Match IPv6 prefix length";
+                value 101;
+              }
+              enum ipv4-next-hop-prefix-length {
+                description "Match next-hop prefix length";
+                value 102;
+              }
+              enum source-protocol {
+                description "Match prefix length";
```

Comment:
Description does not look like matching with enum.

Reply:
Thank you, it is indeed wrong.